masonry/examples/custom_widget.rs

```diff
@@ -138,7 +138,13 @@ impl Widget for CustomWidget {
 
         let mut text_layout = text_layout_builder.build(&self.0);
         text_layout.break_all_lines(None);
-        text_layout.align(None, Alignment::Start, false);
+        text_layout.align(
+            None,
+            Alignment::Start,
+            AlignmentOptions {
```

Comment:
I think we should use `AlignmentOptions::default()` here. The reason that the other parameters aren't in the options is because that `Default` impl exists (https://github.com/linebender/parley/pull/283#issuecomment-2665296623), and we are using exactly the default value.

Reply:
Should I use `AlignmentOptions::default()` in the label widget too, or just in this example? And do you mean that I should use `AlignmentOptions::default()` *instead* of explicitly specifying the one option, or use `..AlignmentOptions::default()` to allow for future extensibility?

Reply:
Apologies for the abiguity. The first for both. That is, use `AlignmentOptions::default()` as the full parameter in both cases.